docs/internals/sync.md

```diff
@@ -20,25 +20,26 @@ In the context of state sync, "state" refers to:
 6. the grid (LSM data; acquired blocks only)
 7. client replies
 
-However, state sync protocol itself only syncs the first 5.
-6/7 are replicated after state sync protocol, driven by grid repair and the disk scrubber.
+State sync consists of two protocols:
+- [Sync Superblock](./vsr.md#protocol-sync-superblock) (syncs 1-5)
```

Comment:
```suggestion
- [Sync SuperBlock](./vsr.md#protocol-sync-superblock) (syncs 1-5)
```

Reply:
The capitalization of "Superblock" here is deliberate -- I am using the word "superblock" as a noun, not referring to the `SuperBlock` struct.

Reply:
I would prefer we just use the same capitalization as the struct (I think I do this even in our other writing about the SuperBlock).

For example, we also do "TigerBeetle". 😁

Reply:
I changed it, but fwiw I don't think "TigerBeetle" is the same situation. "TigerBeetle" is a company, whereas "tigerbeetle" is an insect. :wink: 

But in the superblock case, `SuperBlock` is an implementation of the "superblock" ("Superblock") concept from file systems.

By analogy, the protocol for repairing client replies is labeled "Protocol: Repair Client Replies", not "Protocol: Repair ClientReplies". "ClientReplies" is an implementation concern, "Client Replies" is a protocol concern. Likewise for the [Ss]uper[Bb]lock. :grin:

Reply:
This is going to be funny for posterity...